### DISCUSSION TITLE
added new measurement on measurement tool

### DISCUSSION THREAD

**Discussion by Roberto8303** [FreeCAD/FreeCAD#16626] [Discussions that will turn in to issues but need refining]
pensavo se fosse possibile aggiungere una modalita di misura dei fori come da disegno.
il nuovo strumento misurazione non permette di sapere la distanza in orizzontale o verticale tra due fori
![458603124_541905265177412_8084989716840612449_n](https://github.com/user-attachments/assets/604bbb4b-387b-45c1-abc9-585dcd653537)
![458779309_493267753601925_4495200864569861537_n](https://github.com/user-attachments/assets/7d33e28a-3388-4690-a4af-713ba01f5023)
![460524920_1684212755712592_8770246560181655572_n](https://github.com/user-attachments/assets/ff2ba217-6764-481b-94ca-47ec8a34740a)

**Comment by luzpaz** [comment]
Please ask on the non-english language subforums: https://forum.freecad.org/viewforum.php?f=11
